packs/data/feat-effects.db/effect-battle-curse.json

```diff
@@ -164,19 +164,23 @@
                 "value": 2
             },
             {
+                "details": "",
```

Comment:
Remove empty details?

Reply:
Extract noise, we should probably fix that in extract.